pkg/sql/schema_changer_test.go

```diff
@@ -1476,6 +1476,9 @@ CREATE TABLE t.test (k INT PRIMARY KEY, v INT);
 
```

Comment:
shouldn't the close be here? why is it closed only in the error case?

also close error should be a Fatal, I think.